proposals/exception-handling/Exceptions-formal-overview.md

```diff
@@ -222,6 +223,11 @@ S;C, labels (catch [t*]) ⊢ instr* : []→[t*]
 S;C, labels [t*] ⊢ caught{a val^n} instr* end : []→[t*]
 ```
 
-## TODO Uncaught Exceptions
+## Uncaught Exceptions
+
+To describe uncaught exceptions we add a new terminal value.
```

Comment:
@rossberg suggested we should avoid 'we': https://github.com/WebAssembly/exception-handling/pull/180#discussion_r955923760

Reply:
Thank you, it seems to have slipped my mind!